### PR TITLE
Fix Motor setAcceleration bug in Python.

### PR DESCRIPTION
**Pull request by songyangZhao**
**Description**
Motor setAcceleration missed device tag parameter in Python controller lib.

### CODE REVIEW

**Review by ygoumaz — CHANGES_REQUESTED** [review]
I confirm that the problem is fixed. I would probably rather add the entry under Python API fixes in the changelog:
https://github.com/cyberbotics/webots/blob/53ff149705cf4d3099ec9cd15b3ef75955d83756/docs/reference/changelog-r2023.md?plain=1#L19-L25

File: docs/reference/changelog-r2023.md [hunk 1]
```diff
@@ -23,6 +23,8 @@ Released on ??
       - Field getters sometimes returning an invalid Field object ([#5633](https://github.com/cyberbotics/webots/pull/5633)).
       - `Field.enableSFTracking` and `Field.disableSFTracking` which were failing ([#5640](https://github.com/cyberbotics/webots/pull/5640)).
       - `Motor.enableForceFeedback` where the `sampling_period` argument was missing ([#5797](https://github.com/cyberbotics/webots/pull/5797)).
+      - `Motor.setAcceleration` was missing the device tag parameter ([#6036](https://github.com/cyberbotics/webots/pull/6036)).
+
```
Comment (ygoumaz) [inline docs/reference/changelog-r2023.md:27]:
```suggestion
```

**Review by ygoumaz — CHANGES_REQUESTED** [review]

**Review by ygoumaz — APPROVED** [review]
Thank you!

**Comment by omichel** [comment]
@songyangZhao: I just invited you to become a member of the webots-committers GitHub group so that it should ease your next contribution (you will be able to commit directly on this repo without forking it and to merge your PRs).

**Pull request merged**